### PR TITLE
Use @types modules instead of typings

### PR DESCRIPTION
**Pull request by sv2dev**
@DorianGrey

### CODE REVIEW

**Comment by DorianGrey** [comment]
Hm... is there any reason to not use the `types`-support of `plugin-typescript` as described [here](https://github.com/frankwallis/plugin-typescript#types)? That would prevent to manually link the particular `index.d.ts` files in `tsconfig.json` (regularly, it's possible to add a `types` entry there, but that one only works with properly with node's resolution mechanism).

**Comment by sv2dev** [comment]
I'll have a look at that. Didn't notice the configuration option yet.

**Comment by sv2dev** [comment]
Hmm ... I tried https://github.com/frankwallis/plugin-typescript#types for jasmine and lodash, but it did not work for me. The typings are not found.

**Comment by DorianGrey** [comment]
Hm... I can reproduce this problem, however, I'm still wondering why it doesn't work fine. This version is also used in the [angular-example](https://github.com/frankwallis/plugin-typescript/tree/master/examples/angular) and (from what I've seen) works fine there. So we're either using it in a wrong way, or it's a bug in the plugin. I've added a corresponding commit [here](https://github.com/flaviait/ng2-jspm-template/pull/32/commits/ca3b40e7242ea29b8579ec186418b3b066cdbee5) for further investigation (we may just cut it off in case we won't get this working).

@frankwallis , do you have an idea if we're using this feature in a wrong way here?

**Comment by frankwallis** [comment]
Sorry for taking so long to respond. I have released a fix for the lodash issue in plugin-typescript@5.2.4.

For the jasmine one you need to add 

```
/// <reference types="jasmine"/>
```

to the top of `todo.service.spec.js` to instruct the plugin to load these types. This is similar to doing `import 'jasmine'` but that would fail at runtime in this situation.

**Comment by DorianGrey** [comment]
Thanks for the update - I've updated the dependencies and added the ref to `todo.service.spec.ts`, and it works fine now.

@svi3c , since I've also had to merge with master, please have a second look at this. LGTM, but ...

File: jspm.config.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 SystemJS.config({
+  browserConfig: {
```
Comment (sv2dev) [inline jspm.config.js]: This should not be inserted here. Do you have a jspm.browser.js?
Seems like I accidentally erased the task from the dev-bundle task: https://github.com/flaviait/ng2-jspm-template/commit/a443e60bdd762742ad70b9dd69d87b05507ee17b#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L266

Comment (DorianGrey) [reply jspm.config.js]: Hm... yes, you did that. Haven't seen that immediately. I've updated the commit to call `npm run touch-jspm-browser` within the `dev-bundle` task, before jspm bundle is started. Removed the `browserConfig` entry as well.

**Pull request merged**